pkg/apis/operator/v1/types.go

```diff
@@ -91,6 +92,8 @@ type InstallationSpec struct {
 	// kubernetesProvider.
 	// +optional
 	FlexVolumePath string `json:"flexVolumePath,omitempty"`
+
+	NodeUpdateStrategy appsv1.DaemonSetUpdateStrategy `json:"nodeUpdateStrategy,omitempty"`
```

Comment:
This field is missing API doc comment. 

The comments get generated into documentation for users, so each field needs one.